content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Accessing HPE Data Fabric S3 Storage from Spring Boot S3 Micro Service
+  deployed in K3S cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: true
+priority: 0
+author: Kiran Kumar Mavatoor
```

Comment:
As author image, you can use your "gravatar". Refer to our guide here for information on how to create a gravatar image for the author. https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#tips-and-trick-using-the-cms-editor

Reply:
done